src/lib/pubkey/ec_group/ec_group.cpp

```diff
@@ -370,11 +370,25 @@ EC_Group::EC_Group(const OID& oid,
                    const BigInt& base_y,
                    const BigInt& order) {
    BOTAN_ARG_CHECK(oid.has_value(), "An OID is required for creating an EC_Group");
+   // TODO(Botan4) change lower bound to 192 bits
    BOTAN_ARG_CHECK(p.bits() >= 128, "EC_Group p too small");
    BOTAN_ARG_CHECK(p.bits() <= 521, "EC_Group p too large");
```

Comment:
Perhaps `#ifdef` on `BOTAN_DISABLE_DEPRECATED_FEATURES` and already raise the bound in Botan 3 (and disable all pcurves below this bound)?